learn/wavemaker-release-notes/v10-6-3.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "WaveMaker 10.6.3 - Release date: 04 February 2021"
+id: ""
+sidebar_label: "Release v10.6.3"
+---
+*WaveMaker released a new update version with some bug fixes.*
+
+## Improvements
+
+---
+
+- Introduced a new option, `Show At`, in the Prefab UI configuration. When using a Prefab in another application or a Prefab, all Prefab elements' UI properties display by default when you select the Prefab element. Note, when you choose a widget from the `Show At` list, the UI property will display when you select that UI property.
+- Included an option for Number and Currency widgets to limit the decimal value entry.
+
+## Bug Fixes
+
+- Fixed an issue related to querying a large string data from DB shortened in the variable's `successCallback()`.
+- Fixed an issue related to the placeholder of the Text widget.
+- Fixed an issue with the dynamic display format for the Text widget.
+- Fixed an issue related to console errors in Prefab when using it multiple times in a Live Form.
+- Fixed an issue with the dropdown disabled property of the Autocomplete widget in DataTable.
+- Fixed an issue related to the AnimationInterval property of the Carousel widget when set through the script.
+- Fixed an issue with missing digits in the Number widget when editing the value.
+- Fixed an issue related to Fields overlapping when updating to Material-2.0.  
+**Note**: Adding fixed heights lead to CSS bloating. Therefore, it loses its responsiveness. We advise not to restrict the grid rows with any strict boundaries.
```

Comment:
CSS bloating needs rewording @lovinahmed

Reply:
Adding Fixed heights lead to a rigid CSS structure.